datalad_container/extractors/tests/test_metalad_container.py

```diff
@@ -0,0 +1,62 @@
+import os.path as op
+import pytest
+import subprocess
+import sys
+from pathlib import Path
+from shutil import which
+from unittest.mock import patch
+
+from datalad.api import (
+    Dataset,
+    clone,
+)
+from datalad.cmd import (
+    StdOutCapture,
+    WitlessRunner,
+)
+from datalad.support.exceptions import CommandError
+from datalad.support.external_versions import external_versions, UnknownVersion
+from datalad.tests.utils_pytest import (
+    SkipTest,
+    assert_in,
+    assert_raises,
+    eq_,
+    ok_exists,
+    with_tempfile,
+    with_tree,
+)
+
+if not external_versions["datalad_metalad"]:
+    raise SkipTest("skipping metalad tests")
+
+# Must come after skiptest or imports will not work
+from datalad_container.extractors.metalad_container import MetaladSingularityInspect
+
+
+class TestMetaladSingularityInspect:
```

Comment:
:+1: 

FYI: I put it in a class to think ahead, assuming that the container extractor will have more functionality someday, also tested in this file. The organization I've used in the past is a putting all of the tests for a function (ie singularity inspect) inside a class, and having a separate class for each function. Given that this is it for now, this could be premature.